gulp-tasks/workbox/workbox-generate-cdn-include-v3.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
```

Comment:
Add the JSDoc description to this file and include at least `@fileoverview` and `@author`

Reply:
Done